apps/ledger-live-mobile/src/locales/en/common.json

```diff
@@ -6973,7 +6973,8 @@
           "title": "Invalid QR Code",
           "desc": "Only scan the QR code displayed on Ledger Live for synching your Ledger Live apps.",
           "info": "For the QR code, open “Settings”, select “General”, and then select “Ledger Sync”, “Turn on sync” and “Show QR”.",
-          "cta": "I understand"
+          "cta": "I understand",
+          "tryAgain": "Try again"
```

Comment:
It should be I understand not try again according to the figma 
the key hasn't be replaced inside the component
![image](https://github.com/user-attachments/assets/3ca7edaf-937e-4ade-8332-092cabe3e3e2)